packages/babel-parser/src/parser/expression.js

```diff
@@ -1237,8 +1235,6 @@ export default class ExpressionParser extends LValParser {
         );
       }
       this.sawUnambiguousESM = true;
-    } else if (!this.hasPlugin("importMeta")) {
-      this.raise(id.start, Errors.ImportCallArityLtOne);
```

Comment:
Since this error isn't used anymore, can you remove it from the `Errors` object?